src/Akka.Persistence.MongoDb/Journal/JournalEntry.cs

```diff
@@ -35,10 +35,12 @@ public class JournalEntry
         [BsonElement("Manifest")]
         public string Manifest { get; set; }
 
-
         [BsonElement("Ordering")]
         public BsonTimestamp Ordering { get; set; }
 
+        [BsonElement("Timestamp")]
+        public BsonTimestamp Timestamp { get; set; }
```

Comment:
We should just use the `Ordering` field instead - we can't back-date the previous journal entries otherwise this will cause total chaos.